extensions/apicurio-registry-avro/runtime/pom.xml

```diff
@@ -41,6 +46,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+    
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+      </dependency>
```

Comment:
Is this now needed for some reason?

Reply:
Yes. We use it to simplify parsing here : https://github.com/quarkusio/quarkus/pull/21618/files#diff-a5c8e493cddd3831ed7120b5c27d8f03412a724cb8ca09f0828a08d2c752012bR88